src/index.js

```diff
@@ -0,0 +1,83 @@
+/* eslint-disable no-plusplus, no-return-assign, no-use-before-define,
+import/extensions, implicit-arrow-linebreak, operator-linebreak  */
```

Comment:
- **_Some eslint rules disabled here can be easily fixed: `no-use-before-define`_**